bot.go

```diff
@@ -620,3 +620,27 @@ func (b *Bot) AnswerCallbackQuery(callback *Callback, response *CallbackResponse
 
 	return nil
 }
+
+// GetFile allows to download a image , video and ... just with a FileId
+func (b *Bot) GetFile(file File) (string, error) {
```

Comment:
The bot API method [getFile](https://core.telegram.org/bots/api#getfile) only accepts/needs a `file_id` and *returns* a `File` object in response.

This go function takes a `File` object (meaning you must construct it manually first) and returns a URL pointing to it on Telegram's servers.

This makes it do something completely different from Telegram's API. I would say that is highly confusing and unwanted.

I think it is better if this function would act like the Telegram API method of the same name, so it would take a file ID and return a File object.

You could introduce an extra helper function which takes a File object and returns the URL for that.